staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -198,6 +198,8 @@ type Config struct {
 	// kube-proxy, services, etc.) can reach the GenericAPIServer.
 	// If nil or 0.0.0.0, the host's default interface will be used.
 	PublicAddress net.IP
+	// The port on which to advertise the apiserver to members of the cluster.
```

Comment:
document default here (secure port is used)

Reply:
This is no longer the case. Should it be?

Reply:
Why do we need both this field and ExternalAddress?

Reply:
Currently, --external-host can be set to a different value than --advertise-address. I suppose this is useful if you want your cluster to connect to the master via a private IP but want swagger to advertise a public IP. If we want to combine them, we probably need a flag deprecation.